webapp/src/components/testing/TestSteps.vue

```diff
@@ -39,12 +40,12 @@
         against a chosen dataset of yours. Below, once you assessed the model,
         you can compare the ground truth and the predicted values
         <div class="flex justify-center mt-4">
-          <CustomButton @click="startTest()"> test </CustomButton>
+          <CustomButton @click="startTest()" data-cy="start-test"> test </CustomButton>
```

Comment:
yeah, I saw that a few times in cypress' examples but I don't like this pattern. it mix test specific code into the rest of the codebase. especially here, it is possible to find a button with a "test" label.
same idea below.